src/ZplBuilder.php

```diff
@@ -99,6 +99,18 @@ public function setHome(float $x, float $y)
         $this->commands[] = sprintf('^LH%f,%f', $this->toDots($x), $this->toDots($y));
     }
 
+    /**
+     * If true, the entire label content will be mirrored across a horizontal axis.
+     * If this command is used multiple times, the last usage will take precedence.
+     *
+     * @param bool $isInvert
+     */
+    public function invertLabelOrientation(bool $isInvert = true) : void
```

Comment:
[NB] The comment says if it's used multiple times the last usage will take precedence, but it looks like it'll always set to what you pass in as the argument. 

Or is this saying that since it's a ZPL command if you pass `true` twice you'll be back where you started?

If it's the second case, I think it would be clearer as two methods `setOrientationInverted()` and `setOrientationNormal()`

Reply:
If you send the `^PO` command it will disregard all previous `^PO` commands and use the last. Meaning, that only the last `^PO` command will be honored. If you send `^POI^POI` it will be inverted. If you send `^PON^POI` it will be inverted. If you send `^POI^PON` it will be normal.

Having two methods like you suggested is probably a good call.

Reply:
Ahh thanks, that makes sense. If you want to keep it as one method and add what you just commented to the documentation that would work too!